compiler/dyno/CMakeLists.txt

```diff
@@ -19,17 +19,50 @@ cmake_minimum_required(VERSION 3.13.4)
 
 project(dyno VERSION 0.1)
 
+# TODO: Options are technically only booleans in CMake, although this still
+# appears to work with strings, but I'm not sure it's correct or best practice
+# to do this way. Check if these option() calls should really be set() instead.
 option(DYNO_CHPL_HOME "The CHPL_HOME environment variable for tests to use")
```

Comment:
Probably best to change it to `set` in this PR if that doesn't mess anything up with CI testing etc.